src/main/resources/processes/FOI_DRAFT_ACCEPTANCE.bpmn

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This stage should just be called FOI_ DRAFT or FOI_CONSIDER_AND_DRAFT

Reply:
I have made this change